README.md

```diff
@@ -45,17 +45,17 @@ TODO
 
 ### Linux
 
-    (cd dlls && make)
-    (cd cl_dll && make)
+    (./waf configure -T release)
+    (./waf)
 
 ### OS X
 
 Nothing here.
 
 ### FreeBSD
 
-    (cd dlls && gmake CXX=clang++ CC=clang)
-    (cd cl_dll && gmake CXX=clang++ CC=clang)
+    (CC=clang CXX=clang++ ./waf configure -T release)
```

Comment:
Any reason to use CC/CXX environment variables here?
And why not use one block:
### Linux/*BSD/OS X
or
### Unix-like